infrastructure/backup/README.md

```diff
@@ -0,0 +1,70 @@
+# Backup system - Velero
+
+[Velero](https://velero.io/) is a backup & migration cloud native software design to perform disaster recovery and migrate resources across cluster.
+
+The main purpose of Velero in CrownLabs is to _mitigate_ the following eventualities:
+- failed/buggy updates
+- unintentional resources deletion
+- catastrophical physical cluster destruction/loss of all masters(*)
+a backup system can be of use.
+
+(*) this would require backups to be stored in a physically different/distant location relatively to the cluster.
+
+Notes:
+- saving backups on the cluster itself could help to mitigate unintentional deletions but it's not a great strategy, since deletion of volumes or related resources could lead in deletion of backups themselves.
```

Comment:
```suggestion
- storing backups on the cluster itself could help to mitigate unintentional resource deletions but it is not a great strategy, since the deletion of volumes or related resources could lead to the deletion of backups themselves.
```